### ISSUE TITLE
Facet by multiple attributes

### ISSUE THREAD

**Issue by metasoarous** [matsengrp/olmsted#90]
Whereas #34 is intended to let you facet by the values of a single attribute, here we'd like to facet by a set of attributes such that we get one column/row per attribute. And example is worth a thousand words:

https://vega.github.io/vega/examples/brushing-scatter-plots/

Not as clear how we're going to solve this one. Does vega allow multi select? How do we make this work with #34?

**Comment by eharkins** [comment]
It seems based on the spec in the above example vs the spec in the example in #91 / #34 that this will need to be a separate spec from that of #34

**Comment by metasoarous** [comment]
Well, you'll need to select which variables you want to facet by, which means you'll need to select multiple of them. The standard html thing to do is this: https://www.w3schools.com/tags/att_select_multiple.asp. But I'm not sure how we'd do this in Vega.

**Comment by eharkins** [comment]
Right. Vega allows you to use [other HTML input types](https://vega.github.io/vega/docs/signals#other-input-types) beyond dropdowns, ranges, etc. What I'm saying is that to "facet by multiple attributes" as we mean it here, is something entirely different from option B we discussed here https://github.com/matsengrp/olmsted/issues/34#issuecomment-432775862, where we are grouping data to separate them according to their value for some attribute.
Doing [this (option A)](https://vega.github.io/vega/examples/brushing-scatter-plots/) seems to not be a version of that same concept, but a separate way of doing it from the perspective of the vega code. That being said, maybe there is a way to do the brushing scatter plots example in a less hardcoded way; they create an explicit number of scales, axes corresponding to the fields they want to plot, and give them names specific to those fields.
